test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -175,6 +175,60 @@ public void InvokeWithBindingData()
             Assert.Equal("123", msgs[0].AsString);
         }
 
+        public class ProgramWithTriggerAndCompoundBindingData
+        {
+            public class Poco
+            {
+                public SubOject prop1 { get; set; }
+                public string xyz { get; set; }
+            }
+
+            public class SubOject
+            {
+                public string xyz { get; set; }
+            }
+
+
+            // BindingData is case insensitive. 
+            // And queue name is normalized to lowercase. 
+            public const string QueueOutName = "qName-{prop1.xyz}";
+            public void Func([QueueTrigger(QueueName)] Poco triggers, [Queue(QueueOutName)] ICollector<string> q)
```

Comment:
prop1 is of type 'SubObject', not string.   So I'd expect 'string prop1' would fail with some sort of type-mismatch. 
But 'SubObject prop1' makes sense.  Is that what you'd expect?


---
In reply to: [114873333](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r114873333) [](ancestors = 114873333)